src/soc/intel/common/block/cse/cse.c

```diff
@@ -1131,11 +1131,18 @@ static void cse_set_state(struct device *dev)
 
 	int send;
 	int result;
+#if 0
 	/*
 	 * Check if the CMOS value "me_state" exists, if it doesn't, then
 	 * don't do anything.
 	 */
 	const unsigned int cmos_me_state = get_uint_option("me_state", UINT_MAX);
```

Comment:
We will not use CMOS anyways on this platform so I would do the following:
```suggestion
	const unsigned int cmos_me_state = get_uint_option("me_state", 1);
```
and remove the `#if 0` stuff

Note cmos_me_state is actually inverted, 1 means disable ME.